src/main/java/jenkins/scm/api/SCMSourceOwners.java

```diff
@@ -69,7 +70,7 @@
         public Iterator<SCMSourceOwner> iterator() {
             Jenkins j = Jenkins.getInstance(); // TODO 1.590+ getActiveInstance
```

Comment:
In this case, using `getActiveInstance` changes behavior: exception instead of graceful fallback with an empty iterator. I'm cleaning up other TODOs that don't change bechavior.